pkg/nodeauth/jwt/node_jwt_authenticator.go

```diff
@@ -6,13 +6,19 @@ import (
 	"encoding/hex"
 	"fmt"
 	"log/slog"
+	"time"
 
 	"github.com/golang-jwt/jwt/v5"
 
 	"github.com/smartcontractkit/chainlink-common/pkg/nodeauth/types"
 	"github.com/smartcontractkit/chainlink-common/pkg/nodeauth/utils"
 )
 
+const (
+	// JWTLeeway defines the time leeway for JWT validation to address clock skew between systems
+	JWTLeeway = 5 * time.Second
```

Comment:
ideally should be configurable 
on HMAC auth I did it in this way : 
- http://github.com/smartcontractkit/cre-internal-common/blob/main/auth/hmac/server.go#L25
- https://github.com/smartcontractkit/cre-internal-common/blob/main/auth/hmac/server.go#L53

Reply:
+1 to make it configurable. Could we also set this to a much lower value (e.g. in the order of milliseconds?), since higher values would probably be more prone to security vulnerabilities